components/producers/checkov/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/go-errors/errors"
```

Comment:
please cleanup the imports, the standard library imports should be first, then the third party ones and finally the imports coming from this repository